modulegen/_template/examples_test.go.tmpl

```diff
@@ -1,17 +1,18 @@
-{{ $entrypoint := Entrypoint }}{{ $lower := ToLower }}{{ $title := Title }}package {{ $lower }}_test
+{{ $entrypoint := Entrypoint }}{{ $image := Image }}{{ $lower := ToLower }}{{ $title := Title }}package {{ $lower }}_test
 
 import (
 	"context"
 	"fmt"
 
+	"github.com/testcontainers/testcontainers-go"
 	"github.com/testcontainers/testcontainers-go/modules/{{ $lower }}"
```

Comment:
Hi @mdelapenya,
This is breaking go mod tidy when applied on an example.
```
go run . new example --name questdb --image "questdb/questdb:latest" --title QuestDB
Error: >> error synchronizing the dependencies: exit status 1
```

When you then try in examples/questdb
```
go mod tidy
go: finding module for package github.com/testcontainers/testcontainers-go/modules/questdb
github.com/testcontainers/testcontainers-go/examples/questdb tested by
        github.com/testcontainers/testcontainers-go/examples/questdb.test imports
        github.com/testcontainers/testcontainers-go/modules/questdb: module github.com/testcontainers/testcontainers-go@latest found (v0.23.0, replaced by ../..), but does not contain package github.com/testcon
tainers/testcontainers-go/modules/questdb
```

Reply:
Oh you're right, the `modules` keyword must be taken from the struct's parentDir 🤦 

Will submit a fix ASAP, thanks for reporting it